README.md

```diff
@@ -7,7 +7,7 @@ Contents:
 |Folder|Summary|Kernel Gateway Mode|
 |------|-------|-------------------|
 |[cf_demo](cf_demo)|Recipe for deploying the scotch_demo microservice to [Cloud Foundry](https://www.cloudfoundry.org/) using the [Python buildpack](https://github.com/cloudfoundry/python-buildpack)|[notebook-http](http://jupyter-kernel-gateway.readthedocs.io/en/stable/http-mode.html)|
-|[all_spark_kernels](docker-stacks/all-spark-kernels)|Repurposes the [jupyter/all-spark-notebook](https://github.com/jupyter/docker-stacks/tree/master/all-spark-notebook) Docker image to run a kernel gateway server instead of Jupyter Notebook server|[notebook-http](http://jupyter-kernel-gateway.readthedocs.io/en/stable/http-mode.html)|
+|[all-spark-kernels](all-spark-kernels)|Repurposes the [jupyter/all-spark-notebook](https://github.com/jupyter/docker-stacks/tree/master/all-spark-notebook) Docker image to run a kernel gateway server instead of Jupyter Notebook server|[jupyter-websocket](http://jupyter-kernel-gateway.readthedocs.io/en/stable/jupyter-websocket.html)|
```

Comment:
The link http://jupyter-kernel-gateway.readthedocs.io/en/stable/jupyter-websocket.html does not work for me.